### PR TITLE
Bump patch version to 2.9.1

### LINKED ISSUE #621: Upstream Update: FreeType 2.9.1
### Package Details
  * Package Name: Freetype
  * Latest available Version: 2.9.1

The above mentioned version is newly released by the upstream project and not yet available within Bincrafters. Please add this version.

### PR DESCRIPTION
**Pull request by uilianries**
closes https://github.com/bincrafters/community/issues/621

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -5,10 +5,6 @@ set(CMAKE_VERBOSE_MAKEFILE TRUE)
 include(conanbuildinfo.cmake)
 conan_basic_setup()
 
-
-add_definitions("-DHB_SCRIPT_ADLAM=1097100397")
```
Comment (uilianries) [inline CMakeLists.txt:9]: @SSE4 This magic number is not working for 2.9.1. How could I obtain such value?

File: conanfile.py [hunk 2]
```diff
@@ -7,13 +7,14 @@
...
     name = "freetype"
-    version = "2.9.0"
+    version = "2.9.1"
     description = "FreeType is a freely available software library to render fonts."
     url = "http://github.com/bincrafters/conan-freetype"
     homepage = "https://www.freetype.org"
     license = "BSD"
+    topics = ("conan", "freetype", "fonts")
     author = "Bincrafters <bincrafters@gmail.com>"
-    exports = ["LICENSE.md", "FindFreetype.cmake"]
```
Comment (uilianries) [inline conanfile.py:16]: freetype provides config.cmake file, also Conan is able to generates the cmake_find_package

**Review by Croydon — COMMENTED** [review]
What is the reasoning behind deleting the FindFreetype.cmake file? Not needed anymore?

**Comment by uilianries** [comment]
it's no longer needed because you could use `cmake_find_package` generator. One file less to be maintained by us.

File: CMakeLists.txt (hunk 1 above)
Comment (SSE4) [reply CMakeLists.txt:9]: we need to remove it anyway and find a proper way to fix it, so just remove for now

**Comment by uilianries** [comment]
Could we merge this one?

**Pull request merged**